develop/dev-guide-use-common-table-expression.md

```diff
@@ -1,26 +1,29 @@
 ---
 title: Common Table Expression
+summary: Introduce the CTE capability of TiDB, which is used to write SQL statements more efficiently.
 ---
 
 # Common Table Expression
 
-Due to the objective complexity of the business, sometimes a single SQL statement as long as 2000 lines is written, which contains a large number of aggregations and multi-level sub-query nesting. Maintaining such SQL is a developer's nightmare.
+In some transaction scenarios, due to application complexity, you may sometimes write a single SQL statement of up to 2,000 lines. The statement probably contains a lot of aggregations and multi-level subquery nesting. Maintaining such an SQL statement can be a developer’s nightmare.
 
-In the previous subsections we have described how to use [views](/develop/dev-guide-use-views.md) to simplify queries and how to use [temporary tables](/develop/dev-guide-use-temporary-tables.md) to cache intermediate query results.
+For more information about simplifying queries, see [Views](/develop/dev-guide-use-views.md). For more details about caching intermediate results, see [Temporary tables](/develop/dev-guide-use-temporary-tables.md) to cache intermediate query results.
 
-In this section, we will introduce the Common Table Expression (CTE) syntax in TiDB, which is a more convenient way to reuse query results.
+This document introduces the Common Table Expression (CTE) syntax in TiDB, which is a more convenient way to reuse query results.
 
-Since version 5.1, TiDB supports ANSI SQL 99 standard CTE and its recursive form, which greatly improves the efficiency of developers and DBAs in writing complex business logic SQL, and enhances the maintainability of the code.
+Since TiDB v5.1, TiDB supports the CTE of the ANSI SQL99 standard and recursion. This helps developers and DBAs more efficiently write SQL statements for complex application logic and makes the code easier to maintain.
```

Comment:
```suggestion
Since TiDB v5.1, TiDB supports the CTE of the ANSI SQL99 standard and recursion. With CTE, you can write SQL statements for complex application logic more efficiently and maintain the code much easier.
```